frontend/src/components/ParamsDefaultFields.tsx

```diff
@@ -72,7 +73,7 @@ const ParamsDefaultFields: React.FC<ParamsDefaultFieldsProps> = ({
         <TextInput
           data-testid={inputProps.id}
           {...inputProps}
-          value={String(inputProps.value)}
+          value={inputProps.value === undefined ? '' : String(inputProps.value)}
```

Comment:
```suggestion
          value={String(inputProps.value ?? '')}
```
Another option to consider.